tests/cnf/ran/powermanagement/internal/tsparams/consts.go

```diff
@@ -11,7 +11,9 @@ const (
 	LabelSuite = "powermanagement"
 	// LabelPowerSaveTestCases is the label for a particular test case.
 	LabelPowerSaveTestCases = "powersave"
-	// PowerSaveTimeout is the timeout being used in powersave tests.
+	// LabelCPUFrequency is the label for frequency tuning test cases.
+	LabelCPUFrequency = "cpu_frequency"
```

Comment:
```suggestion
	LabelCPUFrequency = "cpu-frequency"
```
I've been using hyphens so this is just for consistency

Reply:
Thank you done.